apps/user-ms/src/main/java/it/pagopa/selfcare/user/controller/UserController.java

```diff
@@ -262,6 +259,21 @@ public Uni<Response> createOrUpdate(@Valid CreateUserDto userDto) {
                 .map(ignore -> Response.status(HttpStatus.SC_NO_CONTENT).build());
     }
 
+    @Operation(summary = "The API retrieves users with optional filters in input as query params")
```

Comment:
you could explain with some details the endpoint scope.....the path "/{userId}/..." indicate that you will retrieve only one user but the description says "The API retrieves users ... ". In addition, another reference to user is present "personId", what is the meaning of it? you could add some comments about....